app/otr/lib/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const otrSystemMessages = {
+	USER_JOINED_OTR: 'user_joined_otr',
```

Comment:
Use the same pattern like **en.i18n.json**
```suggestion
	USER_JOINED_OTR: 'User_joined_otr',
	USER_REQUESTED_OTR_KEY_REFRESH: 'User_requested_otr_key_refresh',
	USER_KEY_REFRESHED_SUCCESSFULLY: 'User_key_refreshed_successfully',
```

Reply:
Done!
Thanks!